.circleci/config.yml

```diff
@@ -80,26 +80,6 @@ jobs:
           value: "${VERSION}"
 
   test_lint_unit_pact:
-    executor: builder
```

Comment:
Have you seen the hmpps node executor as used here? https://github.com/ministryofjustice/manage-a-workforce-ui/blob/main/.circleci/config.yml#L22 
Possibly a simpler option than starting a docker instance to run some JavaScript tests and linting?